lib/point_quest/quests/attack_value.ex

```diff
@@ -15,7 +15,7 @@ defmodule PointQuest.Quests.AttackValue do
 
   @impl Ecto.Type
   def cast(_invalid_attack) do
-    {:error, "invalid attack value"}
+    :error
```

Comment:
As discussed in Discord, error return from a cast either needs a keyword list as the second tuple element, or to not return as the tuple form to provide a generic error.

As we don't have a strong column to bind on here, opted for the generic error.